cdap-app-fabric/src/main/java/io/cdap/cdap/internal/app/services/ProgramLifecycleService.java

```diff
@@ -1288,6 +1288,9 @@ private void authorizePipelineRuntimeImpersonation(Map<String, String> userArgs)
   }
 
   private  String decodeUserId(String encodedUserId) {
```

Comment:
Annotate the parameter with Nullable if it can be null. Also remove the extra space after `private`

Reply:
Done.